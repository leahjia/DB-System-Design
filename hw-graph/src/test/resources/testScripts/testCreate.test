# University Map Creation testing file.
# Tests that a graph can be created

# Create a graph
CreateMap map1
