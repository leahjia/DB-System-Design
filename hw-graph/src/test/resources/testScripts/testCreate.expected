# University Map Creation testing file.
# Tests that a UnivMap can be created

# Create a graph
created UnivMap map1
